### PR TITLE
hamronization failed file format input

### PR DESCRIPTION
**Pull request by pimarin**
I modified hamronization summarize which failed to analyse files when only .json or .tsv where provided.

I tries the option `*.{tsv,json}` suggested in the issue [4547])https://github.com/galaxyproject/tools-iuc/issues/4547) but don't work.
I juste filetred with the name given by the `ln` command

FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/hamronization/hamronize_summarize.xml [hunk 1]
```diff
@@ -12,20 +12,19 @@
             #elif $report.is_of_type('json'):
                 #set $ext = 'json'
             #end if
-        ln -s '$report' ./report_${counter}.${ext} &&
+        ln -s '$report' ./sumarrize_report_${counter}.${ext} &&
```
Comment (bernt-matthias) [inline tools/hamronization/hamronize_summarize.xml:15]: Can you add 4 spaces to the indentation?

File: tools/hamronization/hamronize_summarize.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<tool id="hamronize_summarize" name="summarize" version="@TOOL_VERSION@+galaxy1" profile="@PROFILE@"> 
+<tool id="hamronize_summarize" name="summarize" version="@TOOL_VERSION@+galaxy1" profile="@PROFILE@">
```
Comment (bernt-matthias) [inline tools/hamronization/hamronize_summarize.xml:1]: We need to bump tool version(s).

Comment (pimarin) [reply tools/hamronization/hamronize_summarize.xml:1]: check

File: tools/hamronization/hamronize_summarize.xml (hunk 1 above)
Comment (pimarin) [reply tools/hamronization/hamronize_summarize.xml]: check

**Comment by pimarin** [comment]
I fixed the problem, update the tool version.
I didn't change test, which used both .tsv and .json together and worked.

**Comment by hexylena** [comment]
@pimarin it looks like you've got some Shovill changes include. Is there any chance you could remove those first three commits? Then I can merge this.

**Comment by hexylena** [comment]
If you'd be interested, I also added test cases explicitly for this behaviour, if you want to copy those from https://github.com/galaxyproject/tools-iuc/pull/4653, it could be nice to prove this works.

**Comment by pimarin** [comment]
Hi @hexylena,
Sorry I was in holidays, I'll work on the galaxy tools during the end of the week or last week I think !

**Comment by pimarin** [comment]
I think all is good now ?

**Comment by bernt-matthias** [comment]
@pimarin looks like this PR would remove shovill.

File: tools/shovill/test-data/flash.hist [hunk 3]
```diff
@@ -0,0 +1,120 @@
+150	62
```
Comment (bernt-matthias) [inline tools/shovill/test-data/flash.hist:1]: guess the files in `tools/shovil/test-data` also do not belong in this PR?

**Review by bernt-matthias — APPROVED** [review]

**Comment by pimarin** [comment]
@bernt-matthias Thank to your patience !

**Comment by hexylena** [comment]
Fantastic, thanks @pimarin !!
